### PR TITLE
migrated intake subsystem

### PR DESCRIPTION
**Pull request by The-Coding-Kid**
completed migration of intake subsystem, no errors in the code

### CODE REVIEW

**Review by Jolteous — APPROVED** [review]
Looks good, we'll wait for @MostOriginalIGN to review it as well

File: src/main/java/frc/robot/constants/DeviceIDs.java
Comment (MostOriginalIGN) [inline src/main/java/frc/robot/constants/DeviceIDs.java:1]: Will conflict later but should be fine.

**Review by MostOriginalIGN — APPROVED** [review]
Everything looks good otherwise, should be fine.

**Review by flashrun24 — APPROVED** [review]
Ok, I think this is good to merge. @The-Coding-Kid merge when you can.

**Pull request merged**